theano/tensor/var.py

```diff
@@ -244,6 +244,18 @@ def __rdivmod__(self, other):
     def __rpow__(self, other):
         return theano.tensor.basic.pow(other, self)
 
+    # ARITHMETIC - MATRIX MULTIPLICATION
```

Comment:
I'm curious why you place these methods here with its own section?

Is matmul a different class of aritmetic operation?